lib/puppet/defaults.rb

```diff
@@ -1,4 +1,7 @@
+require 'puppet/util/fips'
+
 module Puppet
+  include Puppet::Util::FIPS
```

Comment:
As long as you specify `module_function :fips_enabled?`, then you can take this include out, and just call it like you're doing below.